common/test/acceptance/tests/lms/test_teams.py

```diff
@@ -455,3 +456,199 @@ def test_teams_membership(self):
+        )
+
+    def test_user_can_see_error_message_for_incorrect_data(self):
+        """
+        Scenario: The user should be able to see error message in case of increasing length for required fields.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        When I add text > than 255 characters for name field
+        And I click Create button
+        Then I should see the error message for exceeding length.
+        """
+        self.verify_and_navigate_to_create_team_page()
+        self.create_team_page.create_team()
+
+        # Fill the name field with >255 characters to see validation message.
+        self.create_team_page.value_for_text_field(
+            field_id='name',
+            value='EdX is a massive open online course (MOOC) provider and online learning platform. '
+                  'It hosts online university-level courses in a wide range of disciplines to a worldwide '
+                  'audience, some at no charge. It also conducts research into learning based on how '
+                  'people use its platform. EdX differs from other MOOC platforms, such as Coursera and '
+                  'Udacity, in that it is nonprofit and runs on an open-source software platform. '
+                  'EdX was founded by the Massachusetts Institute of Technology and Harvard '
+                  'University in May 2012. EdX was created for students and institutions that seek '
+                  'to transform themselves through cutting-edge technologies, innovative pedagogy, and '
+                  'rigorous courses. More than 70 schools, nonprofits, corporations, and international '
+                  'organizations offer or plan to offer courses on the edX website. As of 22 October 2014, '
+                  'edX has more than 4 million users taking more than 500 courses online.'
+        )
+
+        self.assertEqual(
+            self.create_team_page.validation_message_text,
+            'Your team could not be created. Check the highlighted fields below and try again.'
+        )
+
+    def test_user_can_create_new_team_successfully(self):
+        """
+        Scenario: The user should be able to create new team.
+        Given I am enrolled in a course with a team configuration and a topic
+        When I visit the Create Team page for that topic
+        Then I should see the Create Team header and form
+        When I fill all the fields present with appropriate data
+        And I click Create button
+        Then I should see teams list page with newly created team.
+        """
+        self.assertEqual(self.browse_teams_page.get_pagination_header_text(), 'Showing 0 out of 0 total')
+        self.verify_and_navigate_to_create_team_page()
+
+        self.fill_create_form()
+        self.create_team_page.create_team()
+
+        self.create_team_page.wait_for_ajax()
```

Comment:
Nit: add the `wait_for_ajax()` call to the `create_team()` method. That way it's not the caller's responsibility to prevent flaky test failures.